src/main/java/com/bulkbuy/enterprise/BulkBuyController.java

```diff
@@ -15,14 +19,35 @@ public String index() {
         return "start";
     }
 
+    /**
+     * Endpoint handler for /newOrder
+     * @return the page newOrder.html
+     */
     @RequestMapping("/newOrder")
-    public String newOrder(){
+    public String newOrder(Model model){
+        Order order = new Order();
+        model.addAttribute(order);
```

Comment:
I've added a model to the newOrder endpoint to allow the Thymeleaf inputs and outputs to work.